pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -555,8 +571,6 @@ func testShouldAlwaysPass(jobName, testName, testSuiteName string) string {
 		testName:      testName,
 		testSuiteName: testSuiteName,
 	}
-	// TODO remove this, but for now we have it to match our current hardcoded constants
-	coordinates.testSuiteName = ""
```

Comment:
Nervous about this but it does seem correct. We don't normally define a testSuite before this, now we do and this value should come through. I'm unclear why we were setting empty string prior to this patch though, it should be that by default.